tests/tekton-resources/tasks/generators/clusterloader/load-pod-identity.yaml

```diff
@@ -175,6 +178,68 @@ spec:
       S3_RESULT_PATH=$(cat $(results.s3_result.path))
       echo "S3 Path: $S3_RESULT_PATH"
       aws sts get-caller-identity
+
+      REGION=$(params.region)
+      ENDPOINT_FLAG=""
+      if [ -n "$(params.endpoint)" ]; then
+        ENDPOINT_FLAG="--endpoint $(params.endpoint)"
+      fi
+
+      CLUSTER_NAME=$(params.cluster-name)
+      NAMESPACE="EKSPodIdentityScalabilityTests"
+      DIMENSION_NAME="ClusterName"
+      DIMENSION_VALUE=$CLUSTER_NAME
+      METRIC_LATENCY_NAME="CredentialFetchLatency"
+      PERIOD=300
+
+      START_TIME=$(aws eks $ENDPOINT_FLAG --region $REGION describe-cluster \
```

Comment:
could you add comments on why you consider this as start time ?

Also, please add comments overall to make it more readable for future users/consumers on your team, especially wherever you are making assumptions.